sdk/python/kfp/dsl/for_loop.py

```diff
@@ -64,6 +64,25 @@ def _get_subvar_type(type_name: str) -> Optional[str]:
     return match['value_type'].lstrip().rstrip() if match else None
 
 
+def _get_first_element_type(item_list: ItemList) -> str:
```

Comment:
FYI, no action needed.

The `_` prefix makes sense here since it follows the pattern in the module, but FYI that this mechanism for declaring a symbol private is not needed since it is already in a private module.

We know this is a private module because the public modules are only those included in our [reference docs](https://kubeflow-pipelines.readthedocs.io/en/master/index.html):

![image](https://github.com/kubeflow/pipelines/assets/55268212/a756fc97-7110-49af-a7ed-9f976fb4bfc9)